presto-kafka07/src/main/java/com/facebook/presto/kafka/KafkaConnectorConfig.java

```diff
@@ -76,7 +76,17 @@
     /**
      * Fetch size
      */
-    private int fetchSize = 10 * 1024 * 1024;
+    private DataSize fetchSize = new DataSize(10, Unit.MEGABYTE);
+
+    /**
+     * Default Query Interval
```

Comment:
`defaultQueryInterval` is the variable and `Duration.valueOf("10m")` is the `default` value for this variable which is a `constant` value.